Versions/draft-ietf-dnsop-dns-zone-digest-14.txt

```diff
@@ -730,16 +730,22 @@ Wessels, et al.          Expires April 12, 2021                [Page 13]
 Internet-Draft               DNS Zone Digest                October 2020
 
 
-   2.  For zones that are provably secure, the existence of the apex
-       ZONEMD record MUST be verified.  If the ZONEMD record provably
-       does not exist, digest verification cannot occur.  If the ZONEMD
-       record does provably exist, but is not found in the zone, digest
+       For zones whose DNSKEY RRSet is Insecure or Indeterminate, or if
+       DNSSEC validation is not performed, digest verification continues
+       at step 4 below.  For zones whose DNSKEY RRSet is Bogus, digest
        verification MUST NOT be considered successful.
 
-   3.  For zones that are provably secure, the SOA and ZONEMD RRSets
-       MUST have valid signatures, chaining up to a trust anchor.  If
-       DNSSEC validation of the SOA or ZONEMD records fails, digest
-       verification MUST NOT be considered successful.
+   2.  For zones whose DNSKEY RRSet is Secure, the zone's denial of
+       existence (NSEC/NSEC3) records prove whether or not the apex
+       ZONEMD RRSet exists.  When it is proven to not exist, digest
+       verification cannot occur.  When it is proven to exist, but is
+       not found in the zone, digest verification MUST NOT be considered
+       successful.
+
+   3.  For zones whose DNSKEY RRSet is Secure, the SOA and ZONEMD RRSets
+       MUST have valid signatures.  If DNSSEC validation of the SOA or
+       ZONEMD records fails, digest verification MUST NOT be considered
+       successful.
```

Comment:
Perhaps we could just say that "Normal DNSSEC validation is performed on the SOA and ZONEMD. If the validation of either of these fails, digest verification MUST NOT be considered successful. If validation results in Insecure or Indeterminate (see RFCxxx), digest verification continues at step 4 below." ? It feels like we are unnecessarily describing "normal" DNSSEC behavior, which leads to people poking at it....

Reply:
Yes I was thinking along the same lines earlier today.  Let me take another stab at writing it that way.